guizlet/backend-interview
# RISK-26 update dependencies to address security vulnerabilities

Addresses [RISK-26](https://quizlet.atlassian.net/browse/RISK-26) by updating `express` and `nodemon` ([major update](https://github.com/remy/nodemon/releases/tag/v2.0.0) drops support for node 4 and reduces dependency count) dependencies

## Test plan

1. Clear `node_modules`
2. `yarn install`
3. `npm run`
4. Verify "Hello World!" on [localhost](http://localhost:3000/)
5. Make a change to message in index.js: see nodemon output, refresh page to see new message
